cookbook/migration-vuex.md

```diff
@@ -122,30 +119,30 @@ interface State {
 }
 
 export const useAuthUserStore = defineStore('authUser', {
-  // convert to a function
+  // convertir a una función
   state: (): State => ({
     firstName: '',
     lastName: '',
     userId: null
   }),
   getters: {
-    // firstName getter removed, no longer needed
+    // capturador firstName eliminado, ya no es necesario
```

Comment:
I believe that "getter" should not be translated since "capturador" and other synonims are not used when you want to refer to a getter (or at least here in Spain)

Reply:
done